### PR TITLE
added faq section to the readme

### PR DESCRIPTION
**Pull request by a0viedo**
As discussed today, having FAQs on the readme could be really helpful. Tried to cover the most common cases. Suggestions are welcome :smiley: 

/cc @martinheidegger @Sequoia

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,49 @@
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodeschool/discussions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Gitter][gitter-image]][gitter-chat]
 
 discussions
 ===========
 
-need help? or just wanna ask a question? [open an issue](https://github.com/nodeschool/discussions/issues) on this repo!
+Need help? or just wanna ask a question? [open an issue][discussion/issues] on this repo!
+
+NodeSchool organizers can have discussions [over here][organizers]. If you want to create your own NodeSchool workshop, have a look at the [workshoppers repo][workshoppers].
```
Comment (martinheidegger) [inline README.md]: "... can have discussions about how to run nodeschools and improvements of the organization ... " <- good idea?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,49 @@
...
+
+**Disclaimer**: In you open an issue, it's content, title and code formatting may be edited by the repository owners to improve overall understanding of the problem.
+
+# FAQs
+
+## Installation
+
+### How do I know that Node.js installed correctly?
+
+**Windows** - Open the Start Menu and look for an icon that says "Node.js Command Prompt". Type `node` in the terminal and press Enter. If a weird greater symbol (>) appeared in your terminal with a blinking underscore, then you're able to run Node.
```
Comment (martinheidegger) [inline README.md]: This should be a headline section (to make it linkable)

**Comment by martinheidegger** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -1,11 +1,49 @@
...
 
 discussions
 ===========
 
-need help? or just wanna ask a question? [open an issue](https://github.com/nodeschool/discussions/issues) on this repo!
+Need help? or just wanna ask a question? [open an issue][discussion/issues] on this repo!
+
+NodeSchool organizers can have discussions [over here][organizers]. If you want to create your own NodeSchool workshop, have a look at the [workshoppers repo][workshoppers].
+
+**Disclaimer**: In you open an issue, it's content, title and code formatting may be edited by the repository owners to improve overall understanding of the problem.
```
Comment (Sequoia) [inline README.md]: This is superfluous & could be alarming/confusing to new users. I say reduce the friction of getting users to the help forum as much as possible; presenting them with disclaimers is an unneeded barrier imo.  If someone gets mad about an edit to their issue, that bridge can be crossed when it's come to :sun_with_face:

File: README.md [hunk 4]
```diff
@@ -1,11 +1,49 @@
...
+
+# FAQs
+
+## Installation
+
+### How do I know that Node.js installed correctly?
```
Comment (Sequoia) [inline README.md]: I'd opt for `node -v` here, especially if we're not including instructions on exiting the repl
## proposed revision:
From the Start Menu look for an icon that says "Node.js Command Prompt." Launch it, and then at that command prompt enter the following command to output the the version number of your nodejs install:
```
node --version
```
This should return a version number, something like this: `v0.10.25`

Comment (Sequoia) [inline README.md]: This whole thing could really be for linux/windows/mac just with different instructions for launching the shell.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,49 @@
...
+
+### I installed Node.js, now what?
+
+**Windows** - Open the Start Menu and look for an icon that says "Node.js Command Prompt". Once you open it, you could for example type `npm install javascripting -g` to install Javascripting. If you want to install other workshopper, use the same command replacing the module name.
+
+**Linux/OSX** - Open a terminal and type `npm install javascripting -g` to install Javascripting. If you want to install other workshopper, use the same command replacing the module name.
+
+### I installed Node.js but when I type `node` in my terminal it says `'node' is not recognized` (or `node: command not found`)
+
+It's likely that you have a problem with your installation. Go through the installation steps again and if you still have the problem, open an issue telling us exactly which steps did you followed to install Node.
```
Comment (Sequoia) [inline README.md]: :clap: :clap: yay

File: README.md [hunk 6]
```diff
@@ -28,3 +66,11 @@ It may take a day or two for people to respond here. Please don't send multiple
...
 
 It also helps to include your node, npm, git or other CLI command versions, which you can get by running `node -v`, `npm -v` etc.
+
+
+[gitter-image]: https://badges.gitter.im/Join%20Chat.svg
+[gitter-chat]: https://gitter.im/nodeschool/discussions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+[discussion/issues]: https://github.com/nodeschool/discussions/issues
+[organizers]: https://github.com/nodeschool/organizers
+[workshoppers]: https://github.com/nodeschool/workshoppers
+[node-installed-correctly]: https://github.com/nodeschool/discussions#how-do-i-know-that-nodejs-installed-correctly
```
Comment (Sequoia) [inline README.md]: This link doesn't seem to go anywhere :question:

File: README.md (hunk 4 above)
Comment (Sequoia) [inline README.md]: not sure which question but I think we should link [this article](http://www.altifysoftware.com/developing-node-js-in-windows/) which is a detailed walkthru of getting started on windows.

File: README.md [hunk 7]
```diff
@@ -1,11 +1,49 @@
...
+
+It's likely that you have a problem with your installation. Go through the installation steps again and if you still have the problem, open an issue telling us exactly which steps did you followed to install Node.
+
+### I have Node.js installed but when I type `javascripting` in my terminal it shows an error
+
+If you know that Node installed [correctly][node-installed-correctly], then you may have a problem with the installation of the workshopper. Try to reinstall the workshopper. After that, if you still have the problem open an issue with your error and tell us exactly what happened.
+
+## Working on challenges
+
+### I can't find what is wrong!
```
Comment (Sequoia) [inline README.md]: I recommend pushing this to the end and calling it "My question wasn't answered here!" ideally people would read to *the end* of the FAQ before opening an issue :smile:

**Comment by Sequoia** [comment]
Thanks bunches @a0viedo! :pineapple:

**Comment by Sequoia** [comment]
I hope we can eventually/soon migrate this FAQ to the nodeschool site itself-- it will be much more easily discoverable there. To get to this repo, users have to already have decided they want to file a ticket, which is pretty far down the frustration road for many users.

**Comment by a0viedo** [comment]
I'll be wrapping up improvements on this at the end of this week, probably. Also, I haven't started with the INSTALL.md redaction, but already have some interesting resources:

- [Install node and workshopper video](https://www.youtube.com/watch?v=3rQB5fr_5D4) (windows)
- [Install node and workshopper video](https://www.youtube.com/watch?v=4BShP9q0fz8) (Mac)


Anyone have other resources?

**Comment by martinheidegger** [comment]
@a0viedo https://github.com/nodeschool/discussions/wiki/Installation-suggestions

**Comment by a0viedo** [comment]
@martinheidegger I was counting with that, but you know, with the correct screenshot of the Node.js Command Prompt

**Comment by Sequoia** [comment]
oh yeah :+1: to the vids (assuming they're accurate)-- a lot of people learn easier from vid/audio so text + vid casts the widest possible net. :sunflower:

**Comment by martinheidegger** [comment]
@a0viedo haha! indeed. fixed!

**Comment by Sequoia** [comment]
Oh darn we waited too long on this... I'll try to clean up and merge, sorry to not have acted quicker @a0viedo! We have no official PR workflow so...  I say 1 thumb of another "owner" is enough to merge, 2 if we want to get real formal :wink:

**Comment by Sequoia** [comment]
I have migrated the changes to a new branch (I can't commit to @a0viedo's copy) but that requires closing this PR to open a new one (it won't take 2 prs from the same named branch).

**Pull request closed without merging**